src/mongocxx/test/pool.cpp

```diff
@@ -96,6 +97,7 @@ TEST_CASE(
     REQUIRE(interposed.crl_file == crl_file);
     REQUIRE(interposed.weak_cert_validation == allow_invalid_certificates);
 }
+#endif
```

Comment:
Also, will we eventually want a non-SSL builder to test this?  Or is it something we should just test manually from time to time?